engine/src/main/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -62,6 +60,8 @@ class MaterializeWorkflowDescriptorActor() extends Actor with LazyLogging {
 
   override def receive = {
     case MaterializeWorkflow(workflowId, workflowSourceFiles, conf) =>
+      // TODO: This class should be oblivious to backends.
```

Comment:
I don't think we have a ticket for this in Waffle yet. Should we create one? It might involve cleaning up the WorkflowDescriptor and could turn out to be a pretty big change IMO.